loadtest/main.go

```diff
@@ -205,17 +209,43 @@ func run(config Config) {
 		senders := sendersList[i]
 		wg := wgs[i]
 		for _, sender := range senders {
-			go sender()
+			go func() {
+				tx := sender()
+				if tx != "" {
+					telemetry.IncrCounter(1, "load_test_pending_tx_counts")
+					txs = append(txs, tx)
+				}
+			}()
 		}
 		wg.Wait()
 		lastHeight = newHeight
 	}
+
+	if *constant {
+		// sleep 3 seconds wait for transaction to finish
+		time.Sleep(3 * time.Second)
+		for i := 0; i < len(txs); i++ {
+			txResponse := GetTxResponse(txs[i])
+			if txResponse.Tx == nil {
+				// Replace panic with metrics instead
+				panic("transaction not committed")
+			}
+			telemetry.IncrCounter(1, "load_test_committed_tx_counts")
```

Comment:
telemetry doesn't really work since it's not inside the chain, any other recommended way to add the metrics?

Reply:
let's just remove telemetry fro now